src/DoctrineAuditBundle/Reader/AuditReader.php

```diff
@@ -360,22 +411,24 @@ private function filterByObjectId(QueryBuilder $queryBuilder, $id): QueryBuilder
      * @param null|int        $pageSize
      * @param null|string     $transactionHash
      * @param bool            $strict
+     * @param null|DateTime   $startDate
+     * @param null|DateTime   $endDate
      *
      * @throws AccessDeniedException
      * @throws InvalidArgumentException
      *
      * @return QueryBuilder
      */
-    private function getAuditsQueryBuilder(string $entity, $id = null, ?int $page = null, ?int $pageSize = null, ?string $transactionHash = null, bool $strict = true): QueryBuilder
+    private function getAuditsQueryBuilder(string $entity, $id = null, ?int $page = null, ?int $pageSize = null, ?string $transactionHash = null, bool $strict = true, ?DateTime $startDate = null, ?DateTime $endDate = null): QueryBuilder
     {
         $this->checkAuditable($entity);
         $this->checkRoles($entity, Security::VIEW_SCOPE);
 
-        if (null !== $page && $page < 1) {
+        if (null !== $page && $page < 1 && null === $startDate) {
```

Comment:
This means if `$page` is **not** null and is **less than 1** but provided `$startDate` is **not** null, it won't trigger the exception, this is not good.

Reply:
Sorry, misread this.